package.json

```diff
@@ -52,5 +56,8 @@
       "post-merge": "bash ./hooks/post-merge.sh",
       "pre-commit": "npm run precommit"
     }
+  },
+  "dependencies": {
+    "dotenv": "^8.2.0"
```

Comment:
can use this action instead of adding this as a dependency?
https://github.com/marketplace/actions/dotenv-action

Reply:
Dotenv is just for local testing, it doesn't being used in actions. That is handled via gh-secrets.

Reply:
In that case can we move `dotenv` to `devDependencies`?